README.md

```diff
@@ -136,23 +210,13 @@ If you followed the right structure, during mobros packaging you will see the fo
 ![Screenshot from 2021-11-19 11-42-37](https://user-images.githubusercontent.com/84720623/142617267-01fee218-eb5c-4017-9f7e-57c4067c78af.png)
 
 
-### Get Started - Run locally
+### Installing packages <a id="system-detail-installing"/>
 
-In your project copy and execute the following in your terminal:
-
-```
-wget -qO - https://movai-scripts.s3.amazonaws.com/ros-build.bash | bash
-```
-
-#### Did you know ?
+Motivation: As ros component packaging is debian, it has the necesseties of dependency relationing and dependency treeing as a normal programming language, like mentioning a certain version, even if new versions are avaiable. APT (debian's package manager) has the purpose of keeping operating system packages updated, with the intent of installing latest versions possible, preventing the capability of multi-release lines of a component, or frozen dependencies between ros components.
 
-You know that you can install debs simply by mentioning them in the firmware section of the spawner?
+Mobros fullfills this need by calculating the dependency tree of the inputed packages, before giving an ordered list of packages with their candidate versions to apt for it to install them.
 
-![Screenshot from 2021-11-22 15-27-02](https://user-images.githubusercontent.com/84720623/142888504-9f480db3-3ed7-428c-838f-924816b1f989.png)
-
-With your new packages referenced there, you only need to :
-
-```
-movai-cli firmware my-robot
-```
+Mobros already does some automatic conflict handling.
+The current automatic conflict resolutions done are:
+-  if the dependency conflict is with an installed version, and their difference is simply in the build identifier (4 digit of the deb package version), it ignores the installed version. The rationale is that build versions are considered security/bugfixes.
 
```

Comment:
aren't there more automatic resolution or mobros then rely on apt ?

Reply:
There is the potential to make it more smart. Right now on mobros side this is as far as it goes.